### PR TITLE
#165413070 user should comment on articles

### PR DESCRIPTION
**Pull request by Kabalisa**
Description
=======
This feature is about allowing a user to comment on articles, read comments, edit comments he/she created and delete comments he/she created.

Type of change
=======
N/A

How has it been tested
=======
- pull this branch
- run ```npm install```
- run ```npm start``` to start our react app
-login in to authorshaven
-choose an article
-comment on the article
-read other comments
-edit your comment
-delete your comment.

Checklist:
=======
N/A

Pivotal tracker story ID
=======
[#165413070](https://www.pivotaltracker.com/story/show/165413070)

### CODE REVIEW

**Review by AJAkimana — COMMENTED** [review]
Great work @Kabalisa, I would suggest you put those icon images in `images directory` because it is the one that is copied in the building process.

**Review by AJAkimana — APPROVED** [review]
Good work

**Review by Niyitangasam — APPROVED** [review]

File: src/actions/comment.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+export const deleteComment = (id, slug) => dispatch => {
+  dispatch({
+    type: DELETE_COMMENT,
+    payload: http.delete(`/api/v1/article/${slug}/comments/${id}`)
+  });
+};
+
+export const updateComment = (comment, slug, id) => dispatch => {
+  console.log(localStorage.token);
```
Comment (macphilips) [inline src/actions/comment.js]: Remove console.log

File: src/components/articles/viewArticles.jsx [hunk 2]
```diff
@@ -1,40 +1,35 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { PropTypes } from 'prop-types';
-import NavBar from '../common/navBar';
-import ArticleCard from './articleCard';
+import NavBar from '../common/navBar.jsx';
```
Comment (macphilips) [inline src/components/articles/viewArticles.jsx]: Why add the jsx extension?

File: src/assets/scss/_comment.scss [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  position: relative;
+  left: 23%;
+}
+
+.overlap {
+  display: grid;
+  grid-template-areas: 'comment';
+}
+
+.typeComment {
```
Comment (macphilips) [inline src/assets/scss/_comment.scss]: This css class name breaks the convention we have on the codebase. Avoid using camelcase for css class names

**Pull request merged**